tests/std/tests/P0896R4_stream_iterators/test.cpp

```diff
@@ -101,6 +107,78 @@ void test_ostreambuf_iterator(basic_ostream<CharT, Traits>& os) {
 #endif // __cpp_lib_concepts
 }
 
+template <class BidIt, class CharT = typename iterator_traits<BidIt>::value_type, class Traits = regex_traits<CharT>>
+void test_regex_iterator() {
+    using I = regex_iterator<BidIt, CharT, Traits>;
+    STATIC_ASSERT(is_same_v<typename I::difference_type, ptrdiff_t>);
+    STATIC_ASSERT(is_same_v<typename I::iterator_category, forward_iterator_tag>);

```

Comment:
Why?

Reply:
I think I should remove these unrelated stuffs now.